### ISSUE TITLE
Styles provided by the nested pillar (pdf-ui) are not parsed&working on the GovTool wrapper level.

### ISSUE THREAD

**Issue by MSzalowski** [IntersectMBO/govtool#1585]
Issue investigated by @vpaunovic 

## Context & versions

Using the GovTool wrapper with nested pillar (pdf-ui) styles. Versions of tools and dependencies:

GovTool: 1.0.8
pdf-ui: 0.2.9

## Steps to reproduce

1. Prepared the environment with GovTool and pdf-ui.
2. Started the application.

## Actual behavior

Styles provided by the nested pillar (pdf-ui) are not applied correctly on the GovTool wrapper level. For example, the cards displayed are not of the same height. Inspecting the wrapping div around each card in the slider shows that it has a css-0 class, but this class does not contain any styles. Specifically, it should have height: 100% as indicated in the CSS code lines 15-17:
```css
& > div { 
  height: 100%; 
}
```

However, this style is not being applied.

## Expected behavior

The styles from the pdf-ui should be correctly parsed and applied at the GovTool wrapper level. This includes ensuring that all cards have the same height as defined in the CSS.

## Additional information
Upon manually applying the height to the css-0 class, the styles render correctly, and all cards are of equal height. It seems that the styles are not being included properly when the JS and CSS files are built for import.

![image](https://github.com/user-attachments/assets/88f23df9-93df-4b12-8b5c-5f5b508c4288)
![image (1)](https://github.com/user-attachments/assets/8b753cbe-7fb1-4ce1-9215-041c9416bcbb)
![image (2)](https://github.com/user-attachments/assets/e2733756-6948-4311-8c9d-ff52aaf07ee1)
![image (3)](https://github.com/user-attachments/assets/db44dd23-b1e4-46e5-b0c0-12cf8b648858)
![image (4)](https://github.com/user-attachments/assets/d90ffcce-c2b6-4082-a9c3-8451a9e55564)

**Comment by jdyczka** [comment]
As far as I can tell the styles are applied correctly. 
Lines 15-17 relate only to the direct child of `.slick-slide`, which does have `height: 100%`. It's the descendants of this `div` that are shorter.
The `css-0` class is applied to a `div` if it is a MUI `Box` with no props.
cc @vpaunovic

**Comment by vpaunovic** [comment]
@jdyczka  Yes.. I see it now, good catch. It is because we introduced badges to Proposal Card. I will delete now that box wrapper around it and push it as a new PDF version so it can be used in govtool. I will mention you here about it.

CC: @teske00 @MSzalowski

**Comment by vpaunovic** [comment]
@jdyczka This PR solves it. @teske00 Will bump our PDF version to 0.3.0 in the next PR

**Comment by teske00** [comment]
@MSzalowski @jdyczka  Everything is ready now cc @vpaunovic

**Comment by kneerose** [comment]
As per the discussion and status. This ticket is ready to close.
CC: @bosko-m
